packages/components/src/components/hds/theme-switcher/index.ts

```diff
@@ -0,0 +1,108 @@
+/**
+ * Copyright (c) HashiCorp, Inc.
+ * SPDX-License-Identifier: MPL-2.0
+ */
+
+// ------------------------------------------------------------------------------------------
+// IMPORTANT: this is a temporary implementation, while we wait for the design specifications
+// ------------------------------------------------------------------------------------------
+
+import Component from '@glimmer/component';
+import { inject as service } from '@ember/service';
+import { action } from '@ember/object';
+
+import type { HdsDropdownSignature } from '../dropdown/index.ts';
+import type { HdsDropdownToggleButtonSignature } from '../dropdown/toggle/button.ts';
+import type { HdsIconSignature } from '../icon/index.ts';
+import type HdsThemingService from '../../../services/hds-theming.ts';
+import type {
+  HdsThemes,
+  HdsOnSetThemeCallback,
+} from '../../../services/hds-theming.ts';
+
+interface ThemeOption {
+  theme: HdsThemes | undefined;
+  icon: HdsIconSignature['Args']['name'];
+  label: string;
+}
+
+const OPTIONS: Record<HdsThemes, ThemeOption> = {
+  default: { theme: 'default', icon: 'hashicorp', label: 'Default' },
+  system: { theme: 'system', icon: 'monitor', label: 'System' },
+  light: { theme: 'light', icon: 'sun', label: 'Light' },
+  dark: { theme: 'dark', icon: 'moon', label: 'Dark' },
+};
+
+interface HdsThemeSwitcherSignature {
```

Comment:
```suggestion
export interface HdsThemeSwitcherSignature {
```
[Nit] Export the component signature